### DISCUSSION TITLE
Welcome to css Discussions!

### DISCUSSION THREAD

**Discussion by nzakas** [eslint/css#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JulesCSS** [comment]
I recently started learning HTML, CSS and JavaScript.


I created a webpage consisting of mostly links to other webpages arranged in columns. Along the fixed header there are anchor elements linking to each column of the page. The dimensions of the page are; width: 900vw; height: 90vh;, so there is a considerable amount of content spread out horizontally, most of it to the right of the viewport when the page is in it's normal position, which I suppose is contrary to the conventional layout of webpages. When I clicked on one of these links I expected the targeted href element to scroll to the top left of the viewport, but this did not happen, it scrolled only until the targeted href element reached the right hand side of the viewport. I used a work around to get these links working to my desired effect by wrapping each href element in a hidden div the same width of the viewport.

In Excel Vba I would use for example;

Application.Goto Reference:=Range("Z4"), Scroll:=True

or 

activewindow.ScrollColumn = 26
activewindow.ScrollRow = 4 

and the cell Z4 would jump to the top left hand corner.

I created a test webpage of width: 330vw; and height: 900vh; with fifty linked divs, and I found the following;

If the targeted href element is completely within the viewport or partially below the viewport then it will be scrolled vertically to the top of the viewport and it's horizontal position remains unchanged.

If the targeted href element is partially outside the right hand side of the viewport then it will be scrolled to the top of the viewport and to the right hand side of the viewport.

If the targeted href element is completely outside the viewport or partially outside the left hand side of the veiwport then it will be scrolled to the top of the viewport and to the left hand side of the viewport.

Has other member come across this before?
